### ISSUE TITLE
Failed compilation on react-native-web

### ISSUE THREAD

**Issue by bastienJanon** [software-mansion/react-native-svg#2044]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂 i'm now able to display fontawesome icone on react-native-web

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `react-native-svg@13.9.0` for the project I'm working on.
the compilations failed because of this (web only, mobile was ok)
so i thought that can be usefull for other one

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/@fortawesome/react-native-fontawesome/node_modules/react-native-svg/lib/module/ReactNativeSVG.web.js b/node_modules/@fortawesome/react-native-fontawesome/node_modules/react-native-svg/lib/module/ReactNativeSVG.web.js
index 5bb83a4..dac6e80 100644
--- a/node_modules/@fortawesome/react-native-fontawesome/node_modules/react-native-svg/lib/module/ReactNativeSVG.web.js
+++ b/node_modules/@fortawesome/react-native-fontawesome/node_modules/react-native-svg/lib/module/ReactNativeSVG.web.js
@@ -1,11 +1,12 @@
 import * as React from 'react';
 import {
-// @ts-ignore
-unstable_createElement as ucE, createElement as cE } from 'react-native';
+  // @ts-ignore
+  unstable_createElement as cE
+} from 'react-native';
 import SvgTouchableMixin from './lib/SvgTouchableMixin';
-import { resolve } from './lib/resolve';
 import { transformsArrayToProps } from './lib/extract/extractTransform';
-const createElement = cE || ucE;
+import { resolve } from './lib/resolve';
+const createElement = cE;
 const hasTouchableProperty = props => props.onPress || props.onPressIn || props.onPressOut || props.onLongPress;
 const camelCaseToDashed = camelCase => {
   return camelCase.replace(/[A-Z]/g, m => '-' + m.toLowerCase());
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by willgriffiths** [comment]
Same issue for me and patched in the same way. 

For extra context:

The `createElement` export was [removed in 2019](https://github.com/necolas/react-native-web/commit/d4b9f35d3379e3b92b7dfe279d1a9fa58dacecb6). 

We're bundling with vite and it does not like the missing import:
<details><summary>Vite build error</summary>
<p>
[0] Error: Build failed with 1 error:
[0] ../../../node_modules/react-native-svg/lib/module/ReactNativeSVG.web.js:4:31: ERROR: No matching export in "../../../node_modules/react-native-web/dist/index.js" for import "createElement"
[0]     at failureErrorWithLog (/[[REDACTED]]/node_modules/esbuild/lib/main.js:1636:15)
[0]     at /[[REDACTED]]/node_modules/esbuild/lib/main.js:1048:25
[0]     at runOnEndCallbacks (/[[REDACTED]]/node_modules/esbuild/lib/main.js:1471:45)
[0]     at buildResponseToResult (/[[REDACTED]]/node_modules/esbuild/lib/main.js:1046:7)
[0]     at /[[REDACTED]]/node_modules/esbuild/lib/main.js:1058:9
[0]     at new Promise (<anonymous>)
[0]     at requestCallbacks.on-end (/[[REDACTED]]/node_modules/esbuild/lib/main.js:1057:54)
[0]     at handleRequest (/[[REDACTED]]/node_modules/esbuild/lib/main.js:723:19)
[0]     at handleIncomingPacket (/[[REDACTED]]/node_modules/esbuild/lib/main.js:745:7)
[0]     at Socket.readFromStdout (/[[REDACTED]]/node_modules/esbuild/lib/main.js:673:7)
[0]     at Socket.emit (node:events:513:28)
[0]     at addChunk (node:internal/streams/readable:315:12)
[0]     at readableAddChunk (node:internal/streams/readable:289:9)
[0]     at Socket.Readable.push (node:internal/streams/readable:228:10)
[0]     at Pipe.onStreamRead (node:internal/stream_base_commons:190:23) {
[0]   errors: [
[0]     {
[0]       detail: undefined,
[0]       id: '',
[0]       location: {
[0]         column: 31,
[0]         file: '../../../node_modules/react-native-svg/lib/module/ReactNativeSVG.web.js',
[0]         length: 13,
[0]         line: 4,
[0]         lineText: "unstable_createElement as ucE, createElement as cE } from 'react-native';",
[0]         namespace: '',
[0]         suggestion: ''
[0]       },
[0]       notes: [],
[0]       pluginName: '',
[0]       text: 'No matching export in "../../../node_modules/react-native-web/dist/index.js" for import "createElement"'
[0]     }
[0]   ],
[0]   warnings: []
[0] }
</p>
</details>

**Comment by willgriffiths** [comment]
Just saw that this was fixed in main: https://github.com/software-mansion/react-native-reanimated/pull/4352/files#top

🙌

**Comment by efoken** [comment]
Having the same issue within my Next.js app:

```
- wait compiling /_error (client and server)...
- warn ./node_modules/.pnpm/react-native-svg@13.9.0_react-native@0.71.8_react@18.2.0/node_modules/react-native-svg/lib/module/ReactNativeSVG.web.js
Attempted import error: 'createElement' is not exported from 'react-native' (imported as 'cE').

Import trace for requested module:
./node_modules/.pnpm/react-native-svg@13.9.0_react-native@0.71.8_react@18.2.0/node_modules/react-native-svg/lib/module/ReactNativeSVG.web.js
./node_modules/.pnpm/react-native-svg@13.9.0_react-native@0.71.8_react@18.2.0/node_modules/react-native-svg/lib/module/index.js
```

**Comment by UserNobody14** [comment]
Has there been no progress on this?

**Comment by efoken** [comment]
@UserNobody14 this was already fixed with https://github.com/software-mansion/react-native-svg/pull/2027 and released in version 13.10.0

**Comment by WoLewicki** [comment]
As @efoken mentioned, it should already be fixed, so I'll close the issue. Feel free to comment if something is wrong and we can always reopen it then :rocket:
